build.gradle.kts

```diff
@@ -3,7 +3,7 @@ allprojects {
         // Can be used to test a staging repo
         // maven(uri("https://s01.oss.sonatype.org/content/repositories/comseeq-1065"))
         // Can be used to test a release that didn't reach Maven Central yet but it was published @ sonatype
-        // maven(uri("https://s01.oss.sonatype.org/content/groups/public/"))
+        maven(uri("https://s01.oss.sonatype.org/content/groups/public/"))
```

Comment:
We shouldn't need this.

Reply:
I started a Connector SDK Publish build plan from today's develop. 

https://teamcity.seeq-labs.com/buildConfiguration/CRAB_UbuntuConnectorSdkPublish?branch=%3Cdefault%3E&buildTypeTab=overview&mode=builds

This should also run `sq publish`. We may need to close the release when this is finished.

Reply:
It seems the SDK has been published as SNAPSHOT. I'll try to find a workaround to that

Reply:
66.0.0-v202410141803 has been published.